doc/sphinx/source/analysis.txt

```diff
@@ -24,6 +24,7 @@ all FEP simulations. They then make data available (typically, as a
 
    analysis/solvation
    analysis/dihedral
+   analysis/hbond
```

Comment:
remove – no hbond in this PR (I hope)

Reply:
no hbond in this PR, updates to the docs were misplaced and have since been deleted